### ISSUE TITLE
Dev: json-rpc response content-type is incorrect

### ISSUE THREAD

**Issue by neuroscr** [loki-project/loki-core#1161]
**Describe the bug**
The json-rpc response content-type header is incorrect, Session-Desktop fails to work

**To Reproduce**
See below, use dev branch

**Expected behaviour**
```
curl -vvvv -X POST https://storage.seed1.loki.network/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"get_n_service_nodes", "params":{}' -H 'Content-Type: application/json|grep Content-Type'

Content-Type: application/json
```

**Current Behaviour**
```
curl -vvvv -X POST http://public.loki.foundation:38157/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"get_n_service_nodes", "params":{}' -H 'Content-Type: application/json|grep Content-Type'

Content-Type: text/plain
```

**Screenshots or Logs**
If applicable, add screenshots or logs to help explain your problem.

**Additional Information (please complete the following information):**
 - Operating System: `public.loki.foundation:38157`
 -  Local or Remote node: Remote
 - Version of Loki daemon, or most recent commit hash: `public.loki.foundation:38157`

**Comment by jagerman** [comment]
This is fixed by #1185:

```
$ curl -vX POST http://localhost:22023/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"get_n_service_nodes", "params":{}' -H 'Content-Type: application/json' 2>&1 |grep Content-Type
> Content-Type: application/json
< Content-Type: application/json
```

**Comment by jagerman** [comment]
Fixed by #1208.
